app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    _, current_file, new_file_path = command.split()
+    dir_list = new_file_path.split("/")
```

Comment:
`os.path.split()` will be better here since it will correctly split on all systems